packages/gatsby/src/redux/reducers/last-action.ts

```diff
@@ -0,0 +1,6 @@
+import { IGatsbyState, ActionsUnion } from "../types"
+
+export const lastAction = (
+  _state: IGatsbyState["lastAction"],
```

Comment:
Can you remove the `_` before `state` and re-add the default value?

Reply:
Changed!
Just to explain: I had made this change because the `state` parameter was not used in the function and the code editor was complaining about it. 

Original code:
![Screen Shot 2020-05-05 at 09 19 00](https://user-images.githubusercontent.com/5255042/81067267-cd6dbd00-8eb4-11ea-9c19-b68529244dfc.png)

Circle CI: 
![Screen Shot 2020-05-05 at 09 39 39](https://user-images.githubusercontent.com/5255042/81067595-45d47e00-8eb5-11ea-81c4-ece617b2f392.png)

Proposed code:
![Screen Shot 2020-05-05 at 09 19 25](https://user-images.githubusercontent.com/5255042/81067614-4cfb8c00-8eb5-11ea-8de4-5a5a2368d353.png)

Reply:
Ah, okay, that makes sense. Sorry for not checking it out locally. It makes sense then to have this change. Sorry for the trouble :)

Reply:
Ok, no problem. I reverted the code to my previous change.